subxt/src/error/dispatch_error.rs

```diff
@@ -159,6 +163,12 @@ impl ModuleError {
     pub fn raw(&self) -> RawModuleError {
         self.raw
     }
+
+    /// attempts to decode the ModuleError into a value implementing the trait `RootError`
```

Comment:
```suggestion
    /// Attempts to decode the ModuleError into a value implementing the trait `RootError`
```

Nit: we should start public documentation with upper case, for code comments this is fine tho